sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -492,10 +492,11 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       val installDir = target.value / "scalajs-bundler-jsdom"
       val log = streams.value.log
       val jsdomVersion = (version in installJsdom).value
+      val usingYarn = useYarn.value
       if (!installDir.exists()) {
         log.info(s"Installing jsdom in ${installDir.absolutePath}")
         IO.createDirectory(installDir)
-        install(installDir, useYarn.value, log)(s"jsdom@$jsdomVersion")
+        install(installDir, usingYarn, log)(s"jsdom@$jsdomVersion")
```

Comment:
I wish https://github.com/sbt/sbt/issues/3299 could be fixed soon so that we don’t have to introduce these redundant identifiers (ie `usingYarn` instead of `useYarn.value`)

Reply:
4b2d2f9 updates to newly released sbt 1.0.2 and allows to remove these workarounds.